src/Conda.jl

```diff
@@ -70,7 +70,16 @@ function _set_conda_env(cmd)
         pop!(env, var)
     end
     env["CONDARC"] = CONDARC
-    setenv(cmd, env)
+    env
+end
+
+"""
+Use a cleaned up environment for the command `cmd`.
+
+Any environment variable starting by CONDA will interact with the run.
+"""
+function _set_conda_env(cmd)
```

Comment:
Yes, go for it! I just do not like to have two very similar functions with close names :smiley: